Sources/ArcGISToolkit/Components/Offline/Utilities/URL+Offline.swift

```diff
@@ -15,35 +15,12 @@
 import ArcGIS
 import Foundation
 
-extension FileManager {
-    /// Calculates the size of a directory and all its contents.
-    /// - Parameter url: The directory's URL.
-    /// - Returns: The total size in bytes.
-    func sizeOfDirectory(at url: URL) -> Int {
-        guard let enumerator = enumerator(at: url, includingPropertiesForKeys: [.fileSizeKey]) else { return 0 }
-        var accumulatedSize = 0
-        for case let fileURL as URL in enumerator {
-            guard let size = try? fileURL.resourceValues(forKeys: [.fileSizeKey]).fileSize else {
-                continue
-            }
-            accumulatedSize += size
-        }
-        return accumulatedSize
-    }
-    
-    /// Returns a Boolean value indicating if the specified directory is empty.
-    /// - Parameter path: The path to check.
-    func isDirectoryEmpty(atPath path: URL) -> Bool {
-        (try? FileManager.default.contentsOfDirectory(atPath: path.path()).isEmpty) ?? true
-    }
-}
-
 extension URL {
     /// The path to the web map directory for a specific portal item.
     /// `Documents/OfflineMapAreas/<Portal Item ID>/`
     /// - Parameter portalItemID: The ID of the web map portal item.
-    private static func portalItemDirectory(forPortalItemID portalItemID: PortalItem.ID) -> URL {
-        URL.documentsDirectory.appending(components: "OfflineMapAreas", "\(portalItemID)/")
+    static private func portalItemDirectory(forPortalItemID portalItemID: PortalItem.ID) -> URL {
+        return .documentsDirectory.appending(components: "OfflineMapAreas", "\(portalItemID)/")
```

Comment:
```suggestion
       .documentsDirectory.appending(components: "OfflineMapAreas", "\(portalItemID)/")
```
Up to you, `return` keyword can be [omitted](https://github.com/swiftlang/swift-evolution/blob/main/proposals/0255-omit-return.md) for single expression functions.

Reply:
This was my suggestion. After omitting the return keyword from single expression functions since that feature was introduced, I've found that I miss it except in cases where the entire function declaration is on a single line. The keyword is an easily identifiable clue that the function returns a value, without having to look for the method's return type, which can be much further to the right. So I've started to include them again.